### PR TITLE
Add a quick guide for setting up the remote spawner

### PR DESCRIPTION
**Pull request by clebergnu**
This Documents some crucial steps for a new user of the remote spawner plugin.

Reference: https://github.com/avocado-framework/avocado/issues/5838

### CODE REVIEW

File: optional_plugins/spawner_remote/README.rst [hunk 1]
```diff
@@ -0,0 +1,37 @@
+=====================
+Remote Spawner Plugin
+=====================
+
+This plugin makes use of remote aexpect sessions (to remote hosts or
```
Comment (richtja) [inline optional_plugins/spawner_remote/README.rst:5]: Can you please add a link to aexpect repository, users might not be aware of it and a link could help here.

**Review by richtja — CHANGES_REQUESTED** [review]
Hi @clebergnu, thank you for this update. I have just one proposal about aexpect, what do you think?

File: optional_plugins/spawner_remote/README.rst [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+It mainly draws inspiration and mimics slot and other
+code from the LXC spawner with the exception of extra configurable
+test timeout that is enforced by aexpect as a dependency and fully
+specific to this type of spawner.
+
+To install the Remote Spawner plugin from pip, use::
+
+    $ sudo pip install avocado-framework-plugin-spawner-remote
+
+After installed, add the slot to your avocado config file::
```
Comment (pevogam) [inline optional_plugins/spawner_remote/README.rst:18]: *After it is installed*

Comment (pevogam) [reply optional_plugins/spawner_remote/README.rst:18]: Add *a slot (e.g. "board")* to your avocado config file

**Review by pevogam — APPROVED** [review]
Great @clebergnu, I have to confess I did not expect this extra step on your side but the documentation is both succinct and extensive and other than a few minor typos I say LGTM.

**Comment by clebergnu** [comment]
updated with @richtja and @pevogam comments/requests.

**Review by richtja — APPROVED** [review]
LGMT, thanks for the update.

**Pull request merged**